### ISSUE TITLE
Login failure

### ISSUE THREAD

**Issue by bubalazi** [floydhub/floyd-cli#175]
Hi,

I went on to `pip install -U floyd-cli` on both, python2.7 and python3.5 and attempted to login with no success.

## python2.7.12
```
$ floyd login -u <USER>
Traceback (most recent call last):
  File "/usr/local/bin/floyd", line 7, in <module>
    from floyd.main import cli
  File "/usr/local/lib/python2.7/dist-packages/floyd/main.py", line 11, in <module>
    from floyd.cli.version import upgrade, version, auto_upgrade
  File "/usr/local/lib/python2.7/dist-packages/floyd/cli/version.py", line 1, in <module>
    import pip
  File "/usr/local/lib/python2.7/dist-packages/pip/__init__.py", line 45, in <module>
    from pip.vcs import git, mercurial, subversion, bazaar  # noqa
  File "/usr/local/lib/python2.7/dist-packages/pip/vcs/mercurial.py", line 9, in <module>
    from pip.download import path_to_url
  File "/usr/local/lib/python2.7/dist-packages/pip/download.py", line 40, in <module>
    from pip._vendor import requests, six
  File "/usr/local/lib/python2.7/dist-packages/pip/_vendor/requests/__init__.py", line 98, in <module>
    from . import packages
  File "/usr/local/lib/python2.7/dist-packages/pip/_vendor/requests/packages.py", line 12, in <module>
    sys.modules['pip._vendor.requests.packages.' + mod] = sys.modules["pip._vendor." + mod]
KeyError: 'pip._vendor.urllib3.contrib.pyopenssl'
```
## python3.5.2
```
Traceback (most recent call last):
  File "/media/lap/data/.virtualenvs/new/bin/floyd", line 7, in <module>
    from floyd.main import cli
  File "/media/lap/data/.virtualenvs/new/lib/python3.5/site-packages/floyd/main.py", line 11, in <module>
    from floyd.cli.version import upgrade, version, auto_upgrade
  File "/media/lap/data/.virtualenvs/new/lib/python3.5/site-packages/floyd/cli/version.py", line 1, in <module>
    import pip
  File "/media/lap/data/.virtualenvs/new/lib/python3.5/site-packages/pip/__init__.py", line 45, in <module>
    from pip.vcs import git, mercurial, subversion, bazaar  # noqa
  File "/media/lap/data/.virtualenvs/new/lib/python3.5/site-packages/pip/vcs/mercurial.py", line 9, in <module>
    from pip.download import path_to_url
  File "/media/lap/data/.virtualenvs/new/lib/python3.5/site-packages/pip/download.py", line 40, in <module>
    from pip._vendor import requests, six
  File "/media/lap/data/.virtualenvs/new/lib/python3.5/site-packages/pip/_vendor/requests/__init__.py", line 98, in <module>
    from . import packages
  File "/media/lap/data/.virtualenvs/new/lib/python3.5/site-packages/pip/_vendor/requests/packages.py", line 12, in <module>
    sys.modules['pip._vendor.requests.packages.' + mod] = sys.modules["pip._vendor." + mod]
KeyError: 'pip._vendor.urllib3.contrib'
```
Machine:
Linux home 4.13.0-37-generic #42~16.04.1-Ubuntu SMP Wed Mar 7 16:03:28 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux

**Comment by houqp** [comment]
This is caused by an incompatible change from pip, see: https://github.com/pypa/pip/issues/5081. For the time being, if you can downgrade pip to 9.0.1, that should get it fixed. I am looking into ways to move away from their apis.

**Comment by houqp** [comment]
This is fixed in the latest version (0.10.36), please give it a try :)
